test/celeritas/ext/Vecgeom.test.cc

```diff
@@ -139,11 +139,13 @@ class FourLevelsTest : public VecgeomTestBase
         return this->load_vgdml("four-levels.gdml");
     }
 
+#if VECGEOM_VERSION >= 0x020000
     SpanStringView expected_log_levels() const final
     {
         static std::string_view const levels[] = {"warning"};
         return make_span(levels);
     }
+#endif
```

Comment:
Just use `if (VECGEOM_VERSION >= 0x020000)` so we don't invoke the preprocessor; that's safer because it means the code inside the conditional has to be valid.

so:
```c++
{
if (VECGEOM_VERSION >= 0x020000)
{
    return {};
}
else
{
        static std::string_view const levels[] = {"warning"};
        return make_span(levels);
}
}
```